sysl2/sysl/importer/grammar_test.go

```diff
@@ -0,0 +1,44 @@
+package importer
```

Comment:
There is already a standard test format for this package, use that by just adding another test func to the end of importer_tests.go instead of duplicating the test code.